src/gl/metal/Shaders.metal

```diff
@@ -539,27 +539,23 @@ struct PNUVSOutput
 
 struct PNUVSUniforms1
```

Comment:
Should this 1 be changed to 3? Based on the previous code, this number indicates the number of members?